meta_review/views.py

```diff
@@ -0,0 +1,15 @@
+from meta_review.models import Participant
+from django.shortcuts import render
+from django.db.models import Q
+
+
+def index(request):
+    participants = Participant.objects.all().exclude(
+        Q(pos_in=0),
+        Q(neg_in=0),
+        Q(pos_out=0),
+        Q(neg_out=0),
+        Q(punishment=0)
+    )
+    args = {'participants': participants}
+    return render(request, 'meta_review.html', args)
```

Comment:
Here you're rendering `meta_review.html`.

And [here](https://github.com/coala/community/pull/134/files#diff-f2db580a801ecfe0049dc81b9da19c6cR135) you're passing `meta-review/index.html` that could cause an error, IMO.

Reply:
Nope, it doesn't cause any error 😛 The name of template passed in to render method doesn't matter, as long as there is a corresponding file under template directory.

In this case, since `templates/meta_review.html` exists, `return render(request, 'meta_review.html', args)` is fine.